### DISCUSSION TITLE
Make the grapesjs UI html configurable

### DISCUSSION THREAD

**Discussion by burn2delete** [GrapesJS/grapesjs#4938] [Ideas]
Ideally i'd like to use react and build my own interface on top of grapes while still being able to connect my react components to the exposed grapes functionality.

It would be great if we could pass our own html templates to grapes and control the ui.

**Comment by artf** [comment]
That should be already possible, most of the module docs contain the **Customization** section and examples on how to integrate with external frameworks (eg. [Custom Blocks](https://grapesjs.com/docs/modules/Blocks.html#customization), [Custom Assets](https://grapesjs.com/docs/modules/Assets.html#customization), [Custom Modal](https://grapesjs.com/docs/modules/Modal.html#custom-modal))
